databases/mariadb/Portfile

```diff
@@ -104,6 +110,13 @@ if {$subport eq $name} {
                         -DWITH_LIBWRAP:BOOL=ON \
                         -DWITH_SSL:STRING=no
 
+    platform darwin 23 {
+        if {[string match *clang* ${configure.compiler}]} {
+            configure.cflags-append \
+                        -Wno-error=int-conversion
+        }
+    }
```

Comment:
It will still fail on macOS 15 (Darwin 24) when it is available, which could be as soon as a couple weeks away at the Worldwide Developer Conference. Can the fix be generalized to apply to Darwin 23 and greater?

Reply:
This part done.